source/jormungandr/tests/graphical_isochrones_tests.py

```diff
@@ -351,7 +351,7 @@ def test_graphical_isochrones_invalid_datetime(self):
         normal_response, error_code = self.query_no_assert(q)
 
         assert error_code == 400
-        assert 'Unable to parse datetime, year is out of range' in normal_response['message']
+        assert 'Unable to parse datetime, year' in normal_response['message']
```

Comment:
Dunno why the message has changed ? :eyes: 
The python compatibility should not change that ?

Reply:
This is the exception message from the datetime parser, in python 3 they improved the message to provide the value that was "parsed"